[2024-10-03T17:36:51.897+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T17:36:51.909+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:46:03.626759+00:00 [queued]>
[2024-10-03T17:36:51.917+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:46:03.626759+00:00 [queued]>
[2024-10-03T17:36:51.917+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T17:36:51.944+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 11:46:03.626759+00:00
[2024-10-03T17:36:51.949+0530] {standard_task_runner.py:72} INFO - Started process 441 to run task
[2024-10-03T17:36:51.953+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T11:46:03.626759+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpyjt1r0_i']
[2024-10-03T17:36:51.956+0530] {standard_task_runner.py:105} INFO - Job 71: Subtask validate_data
[2024-10-03T17:36:52.009+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:46:03.626759+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T17:36:52.106+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T11:46:03.626759+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T11:46:03.626759+00:00'
[2024-10-03T17:36:52.109+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T17:36:52.195+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T17:36:52.198+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T17:36:52.298+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T17:36:52.483+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T17:36:52.484+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T17:36:52.485+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 2273.95it/s]
[2024-10-03T17:36:52.518+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 59.08it/s]  
[2024-10-03T17:36:52.518+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:00, 86.89it/s]
[2024-10-03T17:36:52.551+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 44.64it/s]
[2024-10-03T17:36:52.558+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 230.19it/s]
[2024-10-03T17:36:52.585+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 167.75it/s]
[2024-10-03T17:36:52.585+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 167.75it/s]
[2024-10-03T17:36:52.612+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 167.75it/s]
[2024-10-03T17:36:52.623+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 167.75it/s]
[2024-10-03T17:36:52.624+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 167.75it/s]
[2024-10-03T17:36:52.624+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 520.68it/s]
[2024-10-03T17:36:53.013+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T17:36:53.805+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T17:36:53.816+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T17:36:53.817+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T11:46:03.626759+00:00, execution_date=20241003T114603, start_date=20241003T120651, end_date=20241003T120653
[2024-10-03T17:36:53.896+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T17:36:53.944+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T17:36:53.965+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
